easybuild/easyblocks/q/qt.py

```diff
@@ -62,7 +63,13 @@ def configure_step(self):
         elif comp_fam in [toolchain.GCC]:  #@UndefinedVariable
             platform = 'linux-g++-64'
         elif comp_fam in [toolchain.INTELCOMP]:  #@UndefinedVariable
-            platform = 'linux-icc-64'
+            if LooseVersion(self.version) >= LooseVersion('4'):
+                platform = 'linux-icc-64'
+            else:
+                platform = 'linux-icc'
+                # fix -fPIC flag (-KPIC is not correct for recent Intel compilers)
+                qmake_conf = os.path.join('mkspecs', platform, 'qmake.conf')
+                apply_regex_substitutions(qmake_conf, [('-KPIC', '-fPIC')])
```

Comment:
true, open a PR for it :grin: 
The error should tell which file it's not finding.